lib/fastlane/actions/badge.rb

```diff
@@ -51,7 +51,15 @@ def self.available_options
                                        env_name: "FL_BADGE_SHIELD",
                                        description: "add a shield to your app icon from shield.io",
                                        optional: true,
-                                       is_string: true)
+                                       is_string: true),
+          FastlaneCore::ConfigItem.new(key: :alpha,
+                                       env_name: "FL_BADGE_ALPHA",
+                                       description: "adds and alpha badge instead of the default beta one",
```

Comment:
Please start the description with a upper case `Adds`, please update all options like this